### ISSUE TITLE
Event loop closed error comes when trying to get multiple requests

### ISSUE THREAD

**Issue by HarsheetKakar** [aio-libs/aiohttp#4694]
🐞 **Describe the bug**

I tried using `gather` instead of `wait` the result was same
💡 **To Reproduce**
```python
import asyncio
import aiohttp

async def fetch(session, url):
    async with session.get(
            url) as resp:
        print(resp.status)
        data = await resp.text()
        return (data)

async def go():
    async with aiohttp.ClientSession() as session:
        return await fetch(session, 'https://www.google.com/')

async def now():
    async with aiohttp.ClientSession() as session:
        return await fetch(session, 'https://www.google.com/')


loop = asyncio.new_event_loop()
coroutine1 = go()
coroutine2 = now()
task1 = loop.create_task(coroutine1)
task2 = loop.create_task(coroutine2)
loop.run_until_complete(asyncio.wait([task1, task2]))
print('task1 result:', task1.result())
print('task2 result:', task2.result())
loop.close()
```

💡 **Expected behavior**

No error

📋 **Logs/tracebacks**

```python-traceback (paste your traceback in the next line)
Traceback (most recent call last):
File "C:\Users\Hitman\AppData\Local\Programs\Python\Python38\lib\asyncio\proactor_events.py", line 116, in del
self.close()
File "C:\Users\Hitman\AppData\Local\Programs\Python\Python38\lib\asyncio\proactor_events.py", line 108, in close
self._loop.call_soon(self._call_connection_lost, None)
File "C:\Users\Hitman\AppData\Local\Programs\Python\Python38\lib\asyncio\base_events.py", line 719, in call_soon
self._check_closed()
File "C:\Users\Hitman\AppData\Local\Programs\Python\Python38\lib\asyncio\base_events.py", line 508, in _check_closed
raise RuntimeError('Event loop is closed')
RuntimeError: Event loop is closed
```

📋 **Your version of the Python**

```console
$ python --version
Python 3.8.2
```

📋 **Your version of the aiohttp/yarl/multidict distributions**

```console
$ python -m pip show aiohttp
Name: aiohttp
Version: 3.6.2
Summary: Async http client/server framework (asyncio)
Home-page: https://github.com/aio-libs/aiohttp
Author: Nikolay Kim
Author-email: fafhrd91@gmail.com
License: Apache 2
Location: c:\users\hitman\appdata\local\programs\python\python38\lib\site-packages
Requires: yarl, chardet, attrs, async-timeout, multidict
Required-by:
```

```console
$ python -m pip show multidict
Name: aiohttp
Version: 3.0.9
Summary: Async http client/server framework (asyncio)
Home-page: https://github.com/aio-libs/aiohttp/
Author: Nikolay Kim
Author-email: fafhrd91@gmail.com
License: Apache 2
Location: d:\projects\asyncio\venv\lib\site-packages
Requires: attrs, chardet, async-timeout, multidict, yarl
Required-by:
```
```console
$ python -m pip show yarl
Name: yarl
Version: 1.4.2
Summary: Yet another URL library
Home-page: https://github.com/aio-libs/yarl/
Author: Andrew Svetlov
Author-email: andrew.svetlov@gmail.com
License: Apache 2
Location: d:\projects\asyncio\venv\lib\site-packages
Requires: idna, multidict
Required-by: aiohttp
```

📋 **Additional context**

**Comment by hh-h** [comment]
Please, provide aiohttp version and error traceback/exception

**Comment by HarsheetKakar** [comment]
> Please, provide aiohttp version and error traceback/exception

Traceback (most recent call last):
  File "C:\Users\Hitman\AppData\Local\Programs\Python\Python38\lib\asyncio\proactor_events.py", line 116, in __del__
    self.close()
  File "C:\Users\Hitman\AppData\Local\Programs\Python\Python38\lib\asyncio\proactor_events.py", line 108, in close
    self._loop.call_soon(self._call_connection_lost, None)
  File "C:\Users\Hitman\AppData\Local\Programs\Python\Python38\lib\asyncio\base_events.py", line 719, in call_soon
    self._check_closed()
  File "C:\Users\Hitman\AppData\Local\Programs\Python\Python38\lib\asyncio\base_events.py", line 508, in _check_closed
    raise RuntimeError('Event loop is closed')
RuntimeError: Event loop is closed

**Comment by HarsheetKakar** [comment]
> Please, provide aiohttp version and error traceback/exception

I have updated them

**Comment by HarsheetKakar** [comment]
this may be out of scope of this issue, but I feel like it has something to do with `ProactorEventLoop` since the code works fine with python 3.7.

**Comment by hh-h** [comment]
aiohttp 3.6.2 doesn't FULLY compatible with python 3.8, especially on Windows. You can try to change the event loop for tests: call `asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())`.

**Comment by HarsheetKakar** [comment]
ok so can I use python 3.7 without changing the code?

**Comment by hh-h** [comment]
I'm not sure that I follow the question. Will it raise an Exception? Try it :) If it everything that you need, feel free to remove the closing loop statement.

Btw, I suggest that you should update multidict and yarl libraries.

**Comment by webknjaz** [comment]
@HarsheetKakar your traceback seems to be incomplete and only contains entries from asyncio outside of aiohttp. We need a complete traceback (there could be a chain of related tracebacks)

**Comment by HarsheetKakar** [comment]
> @HarsheetKakar your traceback seems to be incomplete and only contains entries from asyncio outside of aiohttp. We need a complete traceback (there could be a chain of related tracebacks)

Thank you sir the problem is solved, I think there is some changes in python 3.8 which makes asyncio and aiohttp buggy in windows.
